README.md

```diff
@@ -16,7 +16,9 @@
             - [Theoretical examples](#theoretical-examples)
     * [Styling / theming](#styling---theming)
 - [Tests](#tests)
-- [Data Test Id](#data-test-id)
+  * [Libraries and tools](#libraries-and-tools)
```

Comment:
Unit test files & test-id are also valid for non React / JS stuff. This one, is specific to Javascript / React and should be moved accordingly to the right section

Reply:
Right! Updated!